modules/table/src/integrationTest/java/org/apache/ignite/distributed/ReplicasSafeTimePropagationTest.java

```diff
@@ -153,6 +151,8 @@ public void testSafeTimeReorderingOnLeaderReElection() throws Exception {
 
         RaftGroupService anotherClient = aliveNode.get().raftClient;
 
+        assertThat(anotherClient.refreshLeader(), willCompleteSuccessfully());
```

Comment:
Why do we need to add this assertion? The same question for the other rest

Reply:
Because otherwise leader might not be elected -> maxObservableTimestamp == Long.MaxValue -> command will fail with SafeTimeReorderingException. In case of common logic that will trigger a retry which is expected, however within given particular test we do not test the retry, thus we should carefully prepare the preconditions, meaning leader election.